aspnet/tutorials/nano-server.rst

```diff
@@ -1,8 +1,164 @@
-.. include:: /../common/stub-topic.txt
+.. _nano-server:
 
-|stub-icon| ASP.NET 5 on Nano Server
-====================================
+ASP.NET 5 on Nano Server
+========================
+
+By `Sourabh Shirhatti`
```

Comment:
Hmm. That's weird. My name is already added to https://github.com/aspnet/Docs/blob/master/common/authors.txt#L13

Do you know why it doesn't render correctly?